com.stansassets.plugins-dev-kit/Editor/UIToolkit/SettingsWindow/PackageSettingsWindow.cs

```diff
@@ -82,6 +82,15 @@ protected void AddTab(string label, VisualElement content)
                 throw new ArgumentException($"Tab '{label}' already added", nameof(label));
             }
         }
+        
+        /// <summary>
+        /// Set the flexible growth property of tabs content container
+        /// </summary>
+        /// <param name="styleFloat"></param>
+        protected void ContentContainerFlexGrow(StyleFloat styleFloat)
+        {
+            m_TabsContainer.contentContainer.style.flexGrow = styleFloat;
+        }
```

Comment:
1. I would consider opening m_TabsContainer with an abstraction to change **flexGrow** instead of this method. Because each time when you will need to "open" something - new method should be added to **PackageSettingsWindow**. But this method is not related to **PackageSettingsWindow**, but related to **TabController**.

2. Also I've found this method inside **TabController**, so maybe convert it into a property?
Or at least rename it to action. Because method can't be named as a **noun**, only as **verb** or **verb + noun**.
![image](https://github.com/StansAssets/com.stansassets.plugins-dev-kit/assets/66057791/ff7ace62-3415-4057-ae62-282d075f2902)

I can manage that by myself (because I have time). So please just tell me your opinion on that :)